internal/provider/jwt_issuers.go

```diff
@@ -30,73 +30,70 @@ import (
 ) {
 	resp.Schema = schema.Schema{
-		MarkdownDescription: "Configuration to allow external OIDC providers to issue tokens for use with CC API.",
+		MarkdownDescription: "Configuration to manage external JWT Issuers to access CockroachDB Cloud API via JWT.",
 		Attributes: map[string]schema.Attribute{
 			"id": schema.StringAttribute{
 				Computed: true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.UseStateForUnknown(),
 				},
-				MarkdownDescription: "ID of the API OIDC Configuration.",
+				Description: "The unique identifier of the JWT Issuer resource.",
 			},
-			"issuer": schema.StringAttribute{
+			"issuer_url": schema.StringAttribute{
 				Required:    true,
-				Description: "The issuer of tokens for the API OIDC Configuration. Usually this is a url.",
+				Description: "The URL of the server issuing JWTs.",
 			},
 			"audience": schema.StringAttribute{
 				Required:    true,
-				Description: "The audience that CC API should accept for this API OIDC Configuration.",
+				Description: "The intended audience for consuming the JWT.",
 			},
 			"jwks": schema.StringAttribute{
-				Required:    true,
-				Description: "The JSON Web Key Set used to check the signature of the JWTs.",
+				Optional:    true,
+				Description: "A set of public keys (JWKS) used to verify the JWT.",
 			},
 			"claim": schema.StringAttribute{
 				Optional:    true,
-				Computed:    true,
-				Description: "The JWT claim that should be used as the user identifier. Defaults to the subject.",
+				Description: "Used to identify the user from the external Identity Provider. Defaults to \"sub\".",
 			},
 			"identity_map": schema.ListNestedAttribute{
+				Optional:    true,
+				Description: "A list of mappings to map the external token identity into CockroachDB Cloud.",
 				NestedObject: schema.NestedAttributeObject{
 					Attributes: map[string]schema.Attribute{
 						"token_identity": schema.StringAttribute{
 							Required:    true,
-							Description: "The token value that needs to be mapped.",
+							Description: "Specifies how to fetch external identity from the token claim. A regular expression must start with a forward slash. The regular expression must be in RE2 compatible syntax. For further details, please see https://github.com/google/re2/wiki/Syntax.",
 						},
 						"cc_identity": schema.StringAttribute{
 							Required:    true,
-							Description: "The username (email or service account id) of the CC user that the token should map to.",
-						},
-						"is_regex": schema.BoolAttribute{
-							Optional:    true,
-							Computed:    true,
-							Description: "Indicates that the token_principal field is a regex value.",
+							Description: "Specifies how to map the fetched token identity to an identity in CockroachDB Cloud. In case of a regular expression for token_identity, this must contain a \\1 placeholder for the matched content.",
```

Comment:
I wonder if there will be a problem understanding if the slashes in the placeholder token should be escaped or not?  I noticed that in the docs they show up as `\1` but in the actual source they need to use `\\1`.  I'm not sure the best way to document this.  Maybe @mdlinville can comment?

Reply:
I agree.
@mdlinville did you have a recommendation here?

I'll keep the description here and within the API in sync.

Reply:
I have no idea about this. If you need a second backslash to show up, don't you just escape it? I'm not familiar the nuts and bolts here.

Reply:
The question is how best to document what value the user should use.   The current output will read as this:

```
...for token_identity, this must contain a \1 placeholder for the matched content.
```

But since the value in the string will need to be escaped, the user will actually use `\\1`.  I was suggesting that maybe we should show `\\1` instead.   But asking you if you had guidance on how to make this distinction more clear to the user.

One potential way would be to link them to an example.

Reply:
Updated the field description to call this out. Open to suggestions.